### PR TITLE
[SDL-0189] Scripts for `Restructuring OnResetTimeout` feature

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
ATF Test Scripts to check #[SDL-0189 Restructuring OnResetTimeout](https://adc.luxoft.com/jira/browse/FORDTCN-1948)

This PR is **ready** for review.

### Summary
Commits 7de343968859cda66fad694cdc5e1bc358888139 and 1568d15d1362c9284ca01d0d1c369ddbb519cbf3 are reviewed by Ford
Added not functional update in f2d1e080e5d0874cefc27c9c806f073b86fd54bb
Added the precondition with successful SetInteriorVD for checking RPC with consent

### ATF version
develop

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/API/Restructuring_OnResetTimeout/001_BC_OnResetTimeout_Success_flow.lua [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---------------------------------------------------------------------------------------------------
```
[the remark this answers is not in the document]
Comment (GetmanetsIrina) [reply test_scripts/API/Restructuring_OnResetTimeout/001_BC_OnResetTimeout_Success_flow.lua:1]: SDL Core should be rebased on actual develop branch

**Comment by GetmanetsIrina** [comment]
@dboltovskyi, please review

File: test_scripts/API/Restructuring_OnResetTimeout/001_BC_OnResetTimeout_Success_flow.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (dboltovskyi) [reply test_scripts/API/Restructuring_OnResetTimeout/001_BC_OnResetTimeout_Success_flow.lua:1]: Accepted

File: test_scripts/API/Restructuring_OnResetTimeout/018_deprecated_TTS_UI_OnResetTimeout_Success_flow.lua [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
```
Comment (dboltovskyi) [inline test_scripts/API/Restructuring_OnResetTimeout/018_deprecated_TTS_UI_OnResetTimeout_Success_flow.lua:1]: Please rename with `BC_OnResetTimeout` prefix as other scripts for this feature

**Pull request merged**